### ISSUE TITLE
[10.0] Translation Value not properly updated from Transifex

### ISSUE THREAD

**Issue by yostashiro** [odoo/odoo#14169]
## Impacted versions:

10.0

## Steps to reproduce:

- Install module account_accountant
- Load Japanese through Settings -> Translations -> Load a Translation
- Go to Settings -> Translations -> Application Terms -> Translated Terms, and filter records with `Translation Value contains "ja.po"`

## Current behavior:

There are several strange-looking translation records that appear as:

```
#-#-#-#-#  ja.po (Odoo 9.0)  #-#-#-#-#
<new translation>
#-#-#-#-#  ja.po (Odoo 9.0)  #-#-#-#-#
<old translation>
```

These incorrect translation values cannot be found in Transifex (translation values in Transifex are correct).

## Expected behavior:

Translation values should just show translation from Transifex.

## Video/Screenshot link (optional):

In Odoo:
![image](https://cloud.githubusercontent.com/assets/7766939/20038980/837801d6-a477-11e6-9478-0db7daf20745.png)

In Transifex:
![image](https://cloud.githubusercontent.com/assets/7766939/20038998/cdd8b57c-a477-11e6-848d-e25a604cb307.png)

**Comment by JKE-be** [comment]
Thanks, @mart-e  will take a look on it.

**Comment by mart-e** [comment]
Hello,

These were leftovers from old translation files. They should not have been present in 10.0, it's from a bad forward port, I will remove them.
Thanks

**Comment by mart-e** [comment]
Should be fixed at 7cfbdd57b4c94b7ea44c93829de72337113d5dfe
